### PR TITLE
Cleanup re: #1579.  Fixup namespace conflicts with builtins (file, dir, ...

### PR DESCRIPTION
**Pull request by oxtopus**
...etc.) as well as minor alignment issues

### CODE REVIEW

**Comment by rhyolight** [comment]
:+1:

File: setup.py [hunk 1]
```diff
@@ -504,10 +507,9 @@ def prepareNupicCore(options, platform, bitness):
     # User specified that they have their own nupic.core
     fetchNupicCore = False
   else:
-    nupicCoreReleaseDir, nupicCoreSourceDir = \
-      getDefaultNupicCoreDirectories()
+    nupicCoreReleaseDir, nupicCoreSourceDir = getDefaultNupicCoreDirectories()
 
-  nupicCoreRemote, nupicCoreCommitish = extractNupicCoreTarget()
+  nupicCoreCommitish = extractNupicCoreTarget()
```
Comment (david-ragazzi) [inline setup.py]: @oxtopus And when someone want point to an other fork than numenta?

Comment (david-ragazzi) [inline setup.py]: I will merge, but just take note of this, please..

Comment (oxtopus) [inline setup.py]: `nupicCoreRemote` is/was unused anyway, so it had no effect.

**Pull request merged**